nni/algorithms/compression/v2/pytorch/pruning/tools/base.py

```diff
@@ -13,8 +14,8 @@
 from torch.nn import Module
 from torch.optim import Optimizer
 
-from nni.algorithms.compression.v2.pytorch.base import Pruner, LayerInfo, Task, TaskResult
-from nni.algorithms.compression.v2.pytorch.utils import OptimizerConstructHelper
+from ...base import Pruner, LayerInfo, Task, TaskResult
+from ...utils import OptimizerConstructHelper, Scaling
```

Comment:
why change to relative import?

Reply:
I wish we could move folder `nni/algorithms/compression/v2/pytorch/` to `nni/compression/pytorch/` in `nni v3.0`, so changed the path containing `v2` to relative path.

Reply:
got it